### PR TITLE
improve environment format and update .gitignore

### PR DESCRIPTION
**Pull request by wimo7083**
Was having trouble with the format of the verbose output, this ended up helping.

**Before**
```dos
λ intercept-build -vvvv make
intercept-build: DEBUG: parse_args_for_intercept_build: Raw arguments ['C:\\Users\\William.Montgomery\\AppData\\Local\\Programs\\Python\\Python37-32\\Scripts\\intercept-build', '-vvvv', 'make']
intercept-build: DEBUG: parse_args_for_intercept_build: Parsed arguments: Namespace(append=False, build=['make'], cc='cc', cdb='compile_commands.json', cxx='c++', override_compiler=False, verbose=4)
intercept-build: DEBUG: run_build: run build ['make'], in environment: {'ALIASES': 'C:\\Users\\William.Montgomery\\Desktop\\cmder\\config\\user-aliases.cmd', 'ALLUSERSPROFILE': 'C:\\ProgramData', 'ANSICON': '158x32766 (158x50)', 'ANSICON_DEF': '7', 'APPDATA': 'C:\\Users\\William.Montgomery\\AppData\\Roaming', 'ARCHITECTURE': '64', 'CHOCOLATEYINSTALL': 'C:\\ProgramData\\chocolatey', 'CHOCOLATEYLASTPATHUPDATE': 'Thu May  3 14:35:12 2018', 'CHOCOLATEYTOOLSLOCATION': 'C:\
```

**After**
```dos
λ intercept-build -vvvv make
intercept-build: DEBUG: parse_args_for_intercept_build: Raw arguments ['C:\\Users\\William.Montgomery\\AppData\\Local\\Programs\\Python\\Python37-32\\Scripts\\intercept-build', '-vvvv', 'make']
intercept-build: DEBUG: parse_args_for_intercept_build: Parsed arguments: Namespace(append=False, build=['make'], cc='cc', cdb='compile_commands.json', cxx='c++', override_compiler=False, verbose=4)
intercept-build: DEBUG: run_build: run build ['make'], in environment:
{'ALIASES': 'C:\\Users\\William.Montgomery\\Desktop\\cmder\\config\\user-aliases.cmd',
 'ALLUSERSPROFILE': 'C:\\ProgramData',
 'ANSICON': '158x32766 (158x50)',
 'ANSICON_DEF': '7',
 'APPDATA': 'C:\\Users\\William.Montgomery\\AppData\\Roaming',
 'ARCHITECTURE': '64',
 'CC': 'intercept-cc',
 'CHOCOLATEYINSTALL': 'C:\\ProgramData\\chocolatey',
 'CHOCOLATEYLASTPATHUPDATE': 'Thu May  3 14:35:12 2018',
 'CHOCOLATEYTOOLSLOCATION': 'C:\\tools', ....
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,114 @@
 __pycache__
 .mypy_cache/
 scan_build.egg-info/
+
```
Comment (rizsotto) [inline .gitignore]: Is this relevant change? I would prefer to do without it.

File: libscanbuild/__init__.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 import shlex
 import subprocess
 import sys
+from pprint import pformat
```
Comment (rizsotto) [inline libscanbuild/__init__.py]: Please follow the pattern to be consistent. Standard packages are imported without `from` keyword.

File: .gitignore (hunk 1 above)
Comment (wimo7083) [reply .gitignore]: The build and dist and dist values appear in a git diff  along with some other files depending on the the environment you use. This prevents them from being possible to be added.

Which part of the change would you rather not have?

Comment (rizsotto) [reply .gitignore]: I would just not add this change together the formatting one. :) Keep the PR focused on one thing at the time... You can submit another one with the `.gitignore` change if you think it's a good change.

**Comment by wimo7083** [comment]
Went ahead and did those things.

How are you running your tests and lint locally?

**Comment by rizsotto** [comment]
Thanks for the updates. It looks good to me.
I use virtualenv and `pip install lit pep8`. To run the test set, it's just simple `lit -v tests`.

**Comment by rizsotto** [comment]
Thanks @wimo7083 , and sorry for the slow processing.

**Pull request merged**